components/cam/bld/namelist_files/use_cases/2010_scream_hr.xml

```diff
@@ -0,0 +1,201 @@
+<prescribed_volcaero_cycle_yr>1           </prescribed_volcaero_cycle_yr>
+
+<!-- Ice nucleation mods-->
+<use_hetfrz_classnuc    >.false.</use_hetfrz_classnuc>
+<use_preexisting_ice    >.false.</use_preexisting_ice>
+<hist_hetfrz_classnuc   >.false.</hist_hetfrz_classnuc>
+<micro_mg_dcs_tdep      >.true.</micro_mg_dcs_tdep>
+<microp_aero_wsub_scheme>1</microp_aero_wsub_scheme>
+
+<!-- For Polar mods-->
+<sscav_tuning            >.true.</sscav_tuning>
+<convproc_do_aer         >.true.</convproc_do_aer>
+<convproc_do_gas         >.false.</convproc_do_gas>
+<convproc_method_activate>2</convproc_method_activate>
+<demott_ice_nuc          >.true.</demott_ice_nuc>
+<liqcf_fix               >.true.</liqcf_fix>
+<regen_fix               >.true.</regen_fix>
+<resus_fix               >.true.</resus_fix>
+<mam_amicphys_optaa      >1</mam_amicphys_optaa>
+
+<fix_g1_err_ndrop>.true.</fix_g1_err_ndrop>
+<ssalt_tuning    >.true.</ssalt_tuning>
+
+<!-- For comprehensive history -->
+<history_amwg>.true.</history_amwg>
+<history_aerosol>.true.</history_aerosol>
+<history_aero_optics>.true.</history_aero_optics>
+
+<!-- File for BC dep in snow feature -->
+<fsnowoptics>lnd/clm2/snicardata/snicar_optics_5bnd_mam_c160322.nc</fsnowoptics>
+
+<!-- Radiation bugfix -->
+<use_rad_dt_cosz>.true.</use_rad_dt_cosz>
+
+<!-- Tunable parameters for 72 layer model -->
+<ice_sed_ai>         500.0  </ice_sed_ai>
+<cldfrc_dp1>         0.045D0</cldfrc_dp1>
+<seasalt_emis_scale> 0.85   </seasalt_emis_scale>
+<dust_emis_fact      > 2.05D0    </dust_emis_fact>
+<seasalt_emis_scale  > 0.85      </seasalt_emis_scale>
+<cldfrc2m_rhmaxi>    1.05D0 </cldfrc2m_rhmaxi>
+<effgw_oro>          0.25    </effgw_oro>
+<effgw_beres>        0.4    </effgw_beres>
+<do_tms>             .false.</do_tms>
+<so4_sz_thresh_icenuc  > 0.05e-6   </so4_sz_thresh_icenuc>
+<n_so4_monolayers_pcage>8.0D0 </n_so4_monolayers_pcage>
+<taubgnd                 >2.5D-3 </taubgnd>
+<raytau0                 >5.0D0</raytau0>
+<prc_coef1               >30500.0D0</prc_coef1>
+<prc_exp                 >3.19D0</prc_exp>
+<prc_exp1                >-1.2D0</prc_exp1>
+<rrtmg_temp_fix          >.true.</rrtmg_temp_fix>
+<nucleate_ice_subgrid    >1.2D0</nucleate_ice_subgrid>
+<cld_sed                 >1.0D0</cld_sed>
+<n_so4_monolayers_pcage  > 8.0D0     </n_so4_monolayers_pcage>
+<relvar_fix              > .true.    </relvar_fix>
+
+<!-- Parameters specific to deep convection (default off for HR compset) -->
+<deep_scheme             > off    </deep_scheme>
+<zmconv_c0_lnd           > 0.007  </zmconv_c0_lnd>
+<zmconv_c0_ocn           > 0.007  </zmconv_c0_ocn>
+<zmconv_dmpdz            >-0.7e-3 </zmconv_dmpdz>
+<zmconv_ke               > 5E-6   </zmconv_ke>
+<zmconv_tiedke_add       > 0.8D0  </zmconv_tiedke_add>
+<zmconv_cape_cin         > 1      </zmconv_cape_cin>
+<zmconv_mx_bot_lyr_adj   > 2      </zmconv_mx_bot_lyr_adj>
+
+<!-- Macrophysics/microphysics coupling -->
+<cld_macmic_num_steps    > 6 </cld_macmic_num_steps>
```

Comment:
comment that doesn't require fixing: doesn't subcycling macmic by 6x when our dt is 75 sec mean that macro and micro is being called every 12.5 sec? Seems excessive but I think @bogensch found that we had a bug preventing it from being bigger. He should issue a PR to fix this very soon so let's not worry about this value in the current PR.

Reply:
Switching cld_macmic_num_steps to 1 (which I think we eventually should) may have a beneficial performance impact.

Reply:
@crterai - do you think we can switch macmic_numsteps to 1 by default in this PR?

Reply:
I think with a 75 sec dt we should be fine w/ macmic_num_steps=1.